src/Sync.ts

```diff
@@ -1,15 +1,15 @@
 import { ComponentEvent } from "@egjs/component";
-import Flicking, { EVENTS, Plugin } from "@egjs/flicking";
+import Flicking, { EVENTS, Panel, Plugin } from "@egjs/flicking";
 
 import { SYNC } from "./const";
 import { addClass, removeClass } from "./utils";
 
 interface SyncOptions {
   type: "camera" | "index";
-  synchronizedFlickings: SychronisableFlickingOptions[];
+  synchronizedFlickings: SychronizableFlickingOptions[];
```

Comment:
I wish the names were unified.

Also, I think that the synchronizedFlickings option is an array of flickings, but it is more like an array of options. It might be better to allow flicking for the array type or use a different name.

이름이 통일됐으면 좋겠습니다.

또한 synchronizedFlickings라는 옵션은 flicking 배열일거라 생각이 들지만 옵션 배열에 가깝죠. 배열 타입에 flicking을 허용하거나 이름을 다른 것으로 사용하는게 좋을 것 같습니다.

Reply:
배열에 SychronizableFlickingOptions과 Flicking을 양 쪽이 모두 사용될 수 있게끔 허용하는 것은 사용하는 입장에서 혼란이 생길 수 있을 것 같습니다. 옵션의 이름을 synchronizedFlickingOptions로 변경하였습니다.
Allowing both SychronizableFlickingOptions and Flicking to be used in an array seems to be confusing for users. Name has been unified to synchronizedFlickingOptions.